pkgs/development/python-modules/blspy/default.nix

```
@@ -40,6 +41,12 @@ buildPythonPackage rec {
         fetchSubmodules = true;
       };
     })
+
+    # https://github.com/Chia-Network/bls-signatures/issues/286
+    (fetchpatch {
+      url = "https://github.com/Chia-Network/bls-signatures/commit/797241e9dae1c164c862cbdb38c865d4b124a601.patch";
```

do you mind making this into a PR then taking from the PR. This is only working because of all refs on the upstream repo

---

There is a PR. If I understand you correctly, I'm not sure this would be better since the PR head is mutable. (and quite likely to change)

---

I guess I'm suggesting that the PR ref should be in there as well
```suggestion
      url = "https://github.com/Chia-Network/bls-signatures/pull/287/commits/797241e9dae1c164c862cbdb38c865d4b124a601.patch";
```